components/Blog/blog.scss

```diff
@@ -0,0 +1,61 @@
+$side-nav-bg-color: #f9f9f9;
+$side-nav-color: #c2c2c2;
+$blog-title: rgb(46, 157, 247);
+$blog-component-read-more-color: rgb(6, 91, 159);
+
+.blog-page {
+  display: flex;
+}
+
+.side-nav-blog {
+  align-self: center;
+  width: 15rem;
+
+  a {
+    display: block;
+    color: $side-nav-color;
+    font-size: 1rem;
+    position: relative;
+    background: $side-nav-bg-color;
+    margin-bottom: 0.2rem;
+    margin-right: 1.5rem;
+  }
+}
+
+.blog-component {
+  display: inline-block;
+
+  .title {
+    font-size: 1.5em;
+    font-family: sans-serif;
+  }
+
+  .title:hover {
+    color: $blog-title;
+    cursor: pointer;
+  }
+
+  img {
+    width: 7em;
+    height: 7em;
+    border-radius: 50%;
+    float: left;
+  }
+
+  .content {
+    font-family: Georgia, serif;
+  }
+}
+
+.blog-component::after {
```

Comment:
Is there a reason this is all css and not just a link element? I'd prefer to just have a link

Reply:
@korlando7 It definitely needs to be a link. It needs to be:

1. Available to screenreaders (many if not most still do not read pseudo element content)
2. Accessible by the keyboard (so using a semantic interactive element)

Also 'read more' is not text that is descriptive enough for users of assistive technology such as screenreaders. [Read more about creative informative link text](https://webaim.org/techniques/hypertext/link_text).

Reply:
I created anchor tag for `read more`.. hope it should be able to resolve `Accessiblity` issue